guides/source/routing.md

```diff
@@ -36,6 +36,8 @@ get '/patients/:id', to: 'patients#show'
 
 the request is dispatched to the `patients` controller's `show` action with `{ id: '17' }` in `params`.
 
+NOTE: rails uses snake_case for controller names here, if you have a multiple word controller like `MonsterTrucksController`, you want to use `monster_trucks#show` for example.
```

Comment:
Can you capitalize Rails?

Reply:
sure, once I figure out how to do that, its my first PR ^^

Reply:
Did you make this change locally or in the GitHub UI?

Reply:
github UI

Reply:
Ok go to the "Files changed" tab, then click the "pencil". Make the change and add a new commit.